src/main/java/pages/MainPage.java

```diff
@@ -33,4 +34,9 @@ public Articles getArticles() {
     public Header getHeader() {
         return header;
     }
+
+    public Menu getMenu() {
+        menu.getMenuButton().click();
```

Comment:
Good approach, but I can suggest to think about adding separate 'openMenu' method and call it in tests before calling this getMenu method. In that case you can call 'getMenu' many times without having side effect of opening/closing menu.